shared/common-adapters/overlay/parent-hoc.tsx

```diff
@@ -52,3 +52,16 @@ const OverlayParentHOC = <Props extends {}>(
 }
 
 export default OverlayParentHOC
+
+export const useOverlayParent = () => {
+  const [showingMenu, setShowingMenu] = React.useState(false)
+  const toggleShowingMenu = () => setShowingMenu(!showingMenu)
+  const [_ref, _setRef] = React.useState<React.Component<any> | null>(null)
+  const getAttachmentRef = () => _ref
+  const setAttachmentRef = isMobile
```

Comment:
It'd be nice to get rid of this pair of functions and just use `ref`.

Reply:
I agree - working on making the typing work.